src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -306,12 +308,7 @@ class BaggedMultiResult(
       val correction = Math.pow(inverseSize * norm(predMat(::, i) - meanPrediction(i)), 2)
 
       /* The correction is prediction dependent, so we need to operate on vectors */
-      val variancePerRow: DenseVector[Double] = 0.5 * (arg(::, i) - Math.E * correction)
-
-      /* Impose a floor in case any of the variances are negative (hacked to work in breeze) */
```

Comment:
This is being replaced by https://github.com/CitrineInformatics/lolo/pull/194/files#diff-7c5cef0ff9e4dba1146bc4ff6c4ea36dR368

Reply:
Technically, the complement of `rawSum > 0` is not `negative`. What about `rawSum == 0`?

The comment for `rectifyEstimatedVariance` does not explain why `maxEntry` is used in one case and `-scores.min` in the other case.

Why does below part make sense?
```
val floor = Math.abs(scores.min)
if (floor < 0.0)
```

Reply:
From the docstring:
```
   * If all of the treePredictions are zero, then this will return zero
```
The likelihood that they are non-zero but cancel is infinitesimal.

I'll add to the comments for the others.